pwiz_tools/Skyline/SettingsUI/ViewLibraryPepInfoList.cs

```diff
@@ -304,7 +339,8 @@ public IList<int> Filter(string filterText, string filterCategory)
                         StringComparison.OrdinalIgnoreCase));
                 // Return the elements from the range whose DisplayText actually matches the filter text.
                 return ImmutableList.ValueOf(new RangeList(range).Where(i => _allEntries[i].DisplayText
-                    .StartsWith(filterText, StringComparison.OrdinalIgnoreCase)));
+                    .Contains(filterText, StringComparison.OrdinalIgnoreCase)));
+                    //.StartsWith(filterText, StringComparison.OrdinalIgnoreCase)));
```

Comment:
I'd leave this alone and have a seperate clause for filterText starts with "*", something like

                if (filterText.StartsWith(@"*") && !_continuousFields.Contains(_selectedFilterCategory))
                {
                    // Wildcard
                    var pattern = filterText.Substring(1);
                    return ImmutableList.ValueOf((new RangeList(0, _allEntries.Count-1)).Where(i => _allEntries[i].DisplayText
                        .IndexOf(pattern, StringComparison.OrdinalIgnoreCase)>=0));
                }

Reply:
I removed the wildcard search for this separate "if nothing found" case for now, since this seems to be a special case for peptides, or rather I don't really understand when it is triggered.